scania-digital-design-system/tegel
# fix(core): removing storybook

**Describe pull-request**  

- Removing the Storybook from the core package
- Setting "private" on root package so we do not publish it by accident

**Solving issue**  

-  Storybook is installed on the root level and seems like it is not needed to be in the core package too.
- We might need to tweak the Netlify command for the build, let's see...
- Setting private: true on the topmost package makes it impossible to release it, but still we can use it for installing other packages

**How to test**  
1. Run the branch on your local and see if everything works as before
2. We have to see if Netlify will fail, we might need to update the build command there


**Suggested test steps**


**Additional context**  
Please, let me know if I am missing some info regarding the Storybook in the Core package.
